### PR TITLE
Start replacing with fastapi

### PR DESCRIPTION
**Pull request by BenSouchet**

### CODE REVIEW

**Comment by BenSouchet** [comment]
We need to check if communication (host listening) with softs are okay, mainly photoshop I think

File: openpype/hosts/photoshop/api/extension/CSXS/manifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<ExtensionManifest ExtensionBundleId="com.openpype.PS.panel" ExtensionBundleVersion="1.0.16" Version="7.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<ExtensionManifest ExtensionBundleId="com.openpype.PS.panel" ExtensionBundleVersion="1.0.17" Version="7.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```
Comment (ccaillot) [inline openpype/hosts/photoshop/api/extension/CSXS/manifest.xml:2]: Changement a revert ?

File: openpype/version.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 """Package declaring Pype version."""
 
-__version__ = "3.16.9-quad-1.15.0"
+__version__ = "3.16.9-quad-1.15.1"
```
Comment (ccaillot) [inline openpype/version.py:4]: Changement a revert ?

**Comment by ccaillot** [comment]
All good

**Review by ccaillot — APPROVED** [review]

**Comment by BenSouchet** [comment]
Cleanup done

**Pull request merged**